torchtext/datasets/text_classification.py

```diff
@@ -373,6 +408,122 @@ def AmazonReviewFull(*args, **kwargs):
     return _setup_datasets(*(("AmazonReviewFull",) + args), **kwargs)
 
 
+def _generate_imdb_data(key, extracted_files, vocab, tokenizer, removed_tokens):
+    _data = []
+    for fname in extracted_files:
+        if 'urls' in fname:
+            continue
+        elif key in fname:
+            if 'pos' in fname:
+                txt_iter = iter(tokenizer(row) for row in io.open(fname,
+                                                                  encoding="utf8"))
+                _iter = numericalize_tokens_from_iterator(vocab,
+                                                          txt_iter,
+                                                          removed_tokens)
+                for tokens in _iter:
+                    _data.append((1,
+                                  torch.tensor([token_id for token_id in tokens]).long()))
+            elif 'neg' in fname:
+                txt_iter = iter(tokenizer(row) for row in io.open(fname,
+                                                                  encoding="utf8"))
+                _iter = numericalize_tokens_from_iterator(vocab,
+                                                          txt_iter,
+                                                          removed_tokens)
+                for tokens in _iter:
+                    _data.append((0,
+                                  torch.tensor([token_id for token_id in tokens]).long()))
+    return _data
+
+
+def IMDB(tokenizer=get_tokenizer("basic_english"),
+         root='.data', vocab=None, removed_tokens=[],
+         data_select=('train', 'test')):
```

Comment:
This one doesn't have a validation set, right?

Reply:
no, there is not.